server/src/main/java/org/elasticsearch/index/mapper/MappedFieldType.java

```diff
@@ -140,10 +143,17 @@ public boolean isSearchable() {
         return isIndexed;
     }
 
+    /**
+     * Returns true if the field is indexed.
+     */
+    public final boolean isIndexed() {
```

Comment:
Thanks, the distinction between searchable and indexed is something that we removed a while ago, if I remember correctly, but we ended up using isSearchable in many places where we really meant isIndexed instead and I found it confusing (even if the two meant the same until now). I find it much clearer now with this distinction, and I double checked that you fixed all the places that need to call isIndexed. Great!